src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Check the [Checklist #9](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md), avoid using * for this